indexing-service/src/test/java/org/apache/druid/indexing/common/actions/RetrieveSegmentsActionsTest.java

```diff
@@ -65,7 +65,7 @@ public static void setup() throws IOException
     expectedUnusedSegments.add(createSegment(Intervals.of("2017-10-07/2017-10-08"), UNUSED_V1));
 
     actionTestKit.getMetadataStorageCoordinator()
-                 .commitSegments(expectedUnusedSegments);
+                 .commitSegments(expectedUnusedSegments, null);
```

Comment:
**For follow-up PR**
Since passing `null` is a very common usage right now, it would be better to keep two variants of the new  methods. It would be easier to identify the usages which pass non-null values and we could also avoid passing nulls all over the place.